### PR TITLE
dev-python/jieba-0.38 version bumps

### PR DESCRIPTION
**Pull request by nyim**
use pypi for downloading current version

### CODE REVIEW

File: dev-python/jieba/jieba-0.38.ebuild [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_{3,4,5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Chinese text segmentation: built to be the best Python Chinese word segmentation module."
+
+HOMEPAGE="https://github.com/fxsjy/jieba"
+#SRC_URI="https://github.com/fxsjy/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (JiajunW) [inline dev-python/jieba/jieba-0.38.ebuild]: Just remove this line.

File: dev-python/jieba/jieba-0.38.ebuild [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+SRC_URI="https://pypi.python.org/packages/f6/86/9e721cc52075a07b7d07eb12bcb5dde771d35332a3dae1e14ae4290a197a/${P}.zip -> ${P}.zip"
```
Comment (tolbkni) [inline dev-python/jieba/jieba-0.38.ebuild]: You can use `mirror://pypi` macro

**Comment by nyim** [comment]
use mirror//pypi as distfile URL

File: dev-python/jieba/jieba-0.38.ebuild [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+PYTHON_COMPAT=( python2_7 python3_{3,4,5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Chinese text segmentation: built to be the best Python Chinese word segmentation module."
+
+HOMEPAGE="https://github.com/fxsjy/jieba"
+
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.zip -> ${P}.zip"
```
Comment (tolbkni) [inline dev-python/jieba/jieba-0.38.ebuild:15]: 1. Prefered to use `.tar.gz` than `.zip`
2. `-> ${P}.zip` is needless here

**Comment by JiajunW** [comment]
LGTM :)

@tolbkni I think he uses zip just because there is only zip file on pypi..

**Comment by tolbkni** [comment]
@JiajunW Yes, it looks like this. Merged.

**Pull request merged**